src/lwc/tileViewLwc/tileViewLwc.html

```diff
@@ -0,0 +1,34 @@
+<template>
+	<lightning-card title="Contact Record" icon-name='standard:contact_list'>
+		<div class="slds-card__body_inner">
+			<template if:true={contacts}>
+				<lightning-layout multiple-rows="true">
+					<template for:each={contacts} for:item="contacts">
+						<lightning-layout-item key={contacts.Id} size="3" class="slds-var-p-around_x-small">
+							<div class="demo-only" style="width:230px">
+								<ul class="slds-has-dividers_around-space">
+									<li class="slds-item bear-tile">
+										<article class="slds-tile slds-tile_board">
+											<lightning-tile label={contacts.Name} style="font-size: 20px; font-weight: bold;" href="https://fexleservicesprivatelimi-bb-dev-ed.lightning.force.com/lightning/o/Contact/list?filterName=Recent"></lightning-tile>			
```

Comment:
URL is for increase little functionality.
When we click on a contact name user jumps to the contact tab

Reply:
You have added a hard coded URL, so for any contact it will open the same tab. If you want to increase a functionality like this then the URL should be dynamic so contact tab will open for respective contact.

Reply:
Sir i tried but not done